msticpy/data/queries/localdata/local_data.yaml

```diff
@@ -7,68 +7,58 @@ metadata:
 defaults:
   metadata:
     data_source: 'security_alert'
-  parameters:
```

Comment:
Does validation fail if there is an empty "parameters" element?

Reply:
As parameters is a dictionary (or an "object" in the JSON validation word) and was defined to contain at least 2 required keys (description & type). Since those keys are required, parameters cannot be empty. By removing this requirement, we could have empty parameters, but also parameters without a description or a type.

Happy to change it if you believe that's a better approach.

https://json-schema.org/understanding-json-schema/reference/object.html#id3